plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+from ansible.module_utils.basic import AnsibleModule
+import os
+from python_on_whales.docker_client import DockerClient
+from python_on_whales.exceptions import (
```

Comment:
Imports of third-party libraries need to be guarded (see https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/import.rst#import).